packages/lib/package/components/MazSelect.vue

```diff
@@ -57,7 +57,7 @@
             },
             `--${color}`,
           ]"
-          :style="{ height: `${itemHeight}px` }"
+          :style="{ height: `${itemHeight}px`, width: `${itemWidth}px` }"
```

Comment:
Use a computed to apply the style of this element

```suggestion
          :style="itemsStyle"
```

And something like that

```ts
const itemsStyle = computed(() => {
  const width = typeof props.itemWidth === 'number' ? `${props.itemWidth}px` : undefined
  
  return { height: `${props.itemHeight}px`, width }
})
```